### ISSUE TITLE
Unable to recover funds with seed and derivation path: m/0'

### ISSUE THREAD

**Issue by biafra23** [breadwallet/breadwallet-ios#203]
I am trying to recover access to funds for a friend. 

We have the seed phrase that allows us to restore different address-types (1*, bc1*) with Electrum. We can see transctions up to March 2021 and May 2021 respectively. But my friend is certain that they did make transactions after that. When following the trail of transactions I can find transactions that they say they authorized. But those do not show up in Electrum. 

- Did BRD-Wallet change the derivation path in 2021?
- Did BRD-Wallet do a re-keying in 2021?
- Did BRD-Wallet create a gap in the derivation that is larger than what Electrum assumes?

What else could explain this? 

Slightly off-topic:
Is there a tool out there that can take a seed phrase and then generate key pairs with all publicly documented derivation paths and ignoring gaps (just generate 1 mio addresses). And then scan all these addresses for funds. 

Or: In this case I could provide the address that does have the funds and let this tool run until that address and its private key is generated.

**Comment by acorscadden** [comment]
Hey Dirk,

I haven't used it in a while, but I think Electrum only supports one type of address at a time. BRD derived both segwit and non-segwit addresses from the same seed phrase.

I think you'll need to create two different electrum wallets: one segwit and one non-segwit.

I'm not aware of an address generation tool that you mentioned, but that would be cool if it did.

**Comment by EBGToo** [comment]
The Coinbase wallet (assuming that is an option) surely would allow you to see all your BRD transactions and them move them to a wallet of your choice; rather than struggling with seeds, derivation paths and addresses.

**Comment by biafra23** [comment]
They tried the Coinbase wallet. It does not show latest balance. Assuming it doesn't generate the respective key pairs as well.

**Comment by biafra23** [comment]
> I think you'll need to create two different electrum wallets: one segwit and one non-segwit.

I did that as I mentioned, I can see address types 1* and bc1*.

**Comment by DrrDino** [comment]
> I am trying to recover access to funds for a friend.
> 
> We have the seed phrase that allows us to restore different address-types (1*, bc1*) with Electrum. We can see transctions up to March 2021 and May 2021 respectively. But my friend is certain that they did make transactions after that. When following the trail of transactions I can find transactions that they say they authorized. But those do not show up in Electrum.
> 
> * Did BRD-Wallet change the derivation path in 2021?
> * Did BRD-Wallet do a re-keying in 2021?
> * Did BRD-Wallet create a gap in the derivation that is larger than what Electrum assumes?
> 
> What else could explain this?
> 
> Slightly off-topic: Is there a tool out there that can take a seed phrase and then generate key pairs with all publicly documented derivation paths and ignoring gaps (just generate 1 mio addresses). And then scan all these addresses for funds.
> 
> Or: In this case I could provide the address that does have the funds and let this tool run until that address and its private key is generated.

 will advise you to contact the best hacker around he helped me recover my wallet,fast and reliable his telegram is https://t.me/Blackhathacker111

**Comment by syhd142** [comment]
hello @biafra23 did you resolved this? I have met the same situation too, have tried all method and derivation for m/0' and m/44'/1'/0', feel's not good...

**Comment by biafra23** [comment]
No. I haven't resolved this, yet. My next step would be to download the source around that time and modify it to generate the addresses for me. But this is not urgent (for me :-). So it might take some time until I try to resolve this again.

**Comment by syhd142** [comment]
I have downloaded the breadwallet-ios source code, in the latest version I saw that BRD use this WalletKit(https://github.com/blockset-corp/walletkit/tree/master-brd-app) project as the bitcoin address recovery and generate. I compiled the latest WalletKit source code, I entered my seed phrase, unfortunately, the WalletKit tool shows first unused address is the same as Electrum wallet which have the same derivation path m/0'. I have increased the gap in Electrum but it didn't show any history.

So, I'm doubt whether BRD team has changed the derivation path or WallKit library during the releases. By the way, my first transaction in BRD was about 2017 later to 2018 middle, and my address is legacy, I can even found that in exchange withdraw record, I checked the address, its on the chain, and the coin is there.

**Comment by chiron08** [comment]
so you could restore your bitcoin that way?
